packages/@uppy/react/src/useWebcam.ts

```diff
@@ -0,0 +1,24 @@
+import { useMemo } from 'react'
+import { useSyncExternalStore } from 'use-sync-external-store/shim/index.js'
```

Comment:
why not ? `import { useSyncExternalStore } from 'react';` ? as react 18 onwards supports it ?